website/app/layout.tsx

```diff
@@ -47,6 +47,19 @@ const RootLayout: FC<{
           { children: 'Rules', href: '/rules' },
           { children: 'Playground', href: '/play' },
         ],
+        developerMenu: [
+          {
+            href: '/docs',
+            icon: PaperIcon,
```

Comment:
did you meant ? you cannot pass functions from server to client components
```suggestion
            icon: <PaperIcon />,
```

Reply:
You're right, but we'll have to update Components first.

Reply:
Done